bpf/bpf_lxc.c

```diff
@@ -1009,10 +1009,6 @@ static __always_inline int handle_ipv4_from_lxc(struct __ctx_buff *ctx, __u32 *d
 
 #ifdef ENABLE_EGRESS_GATEWAY
 	{
```

Comment:
nit: now that we don't need anymore to define variables in this block I think we can get rid of these brackets